docker/api/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Defines a simple Argus API server running from a configurable branch or tag
+# in the upstream repo
+FROM python:3.8
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && apt-get install -y --no-install-recommends tini build-essential
+
+# Install argus from configured branch/tag and lock in some dev requirements
+# that aren't part of the regular setup.cfg:
+ARG BRANCH=master
+RUN pip install psycopg2-binary channels==2.4.0 django-extensions python-dotenv git+https://github.com/Uninett/argus@${BRANCH}
```

Comment:
Docker can use `.env`-files, yes. If you do, please document it.

Reply:
I don't, but the `dev` settings which `dockerdev` is based on does. Removing dotenv from argus backend should probably be posted as an issue there.